test/Hpack/UtilSpec.hs

```diff
@@ -38,7 +38,9 @@ spec = do
 
   describe "toModule" $ do
     it "maps paths to module names" $ do
-      toModule ["Foo", "Bar", "Baz.hs"] `shouldBe` Just "Foo.Bar.Baz"
+      toModule ["Foo", "Bar", "Baz.hs"]  `shouldBe` Just "Foo.Bar.Baz"
+      toModule ["Foo", "Bar", "Baz.lhs"] `shouldBe` Just "Foo.Bar.Baz"
+      toModule ["Foo", "Bar", "Baz.hsc"] `shouldBe` Just "Foo.Bar.Baz"
```

Comment:
I think it's a good thing to unit tests this, but this should be 3 test cases, one expectation per test.